.pre-commit-config.yaml

```diff
@@ -0,0 +1,49 @@
+# This file contains 2 stages
+# pre-commit stage - runs by default pre-commit
+# manual stage - runs the same tools but modifies files
+# run as: pre-commit run --all-files --hook-stage manual
+repos:
+  - repo: https://gitlab.cee.redhat.com/infosec-public/developer-workbench/tools.git
+    rev: rh-pre-commit-2.3.0
+    hooks:
+      # If you have not run this hook on your system before, it may prompt you to
+      # log in for patterns, and you will need to try again.
+      #
+      # Docs: https://source.redhat.com/departments/it/it-information-security/leaktk/leaktk_components/rh_pre_commit
+      - id: rh-pre-commit
+        name: Red Hat pre-commit (check)
+      # - id: rh-pre-commit.commit-msg # Optional for commit-msg attestation
+  - repo: https://github.com/PyCQA/docformatter
+    rev: v1.7.7
+    hooks:
+      - id: docformatter
+        name: docformatter (check)
+        additional_dependencies: [tomli]
+        args: [--check, --config, ./pyproject.toml]
+        stages: [pre-commit]
+      - id: docformatter
+        name: docformatter (fix)
+        additional_dependencies: [tomli]
+        args: [--in-place, --config, ./pyproject.toml]
+        stages: [manual]
+  - repo: https://github.com/psf/black
+    rev: 25.1.0
+    hooks:
+      - id: black
+        name: black (check)
+        args: [--check]
+        stages: [pre-commit]
+      - id: black
+        name: black (fix)
+        stages: [manual]
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.12.1
+    hooks:
+      # Run the linter
```

Comment:
This code comment seems superfluous (e.g., there is no analog for it before line 32...).

Reply:
Fixed. This was there when `ruff` was used for both linting and formatting.